src/logger.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
+const fs = require("fs");
+const path = require("path");
+const {performance} = require('perf_hooks');
+
+const date = new Date();
+
+const fileName = `${date.getFullYear()}-${date.getMonth()}-${date.getDay()}-log`
+
+const pathLogDir = path.join(process.env.HOME, "EventsExpressNode-logs");
+const pathLogFile = path.join(pathLogDir, fileName);
+
+const isAccessible = path => {
+  return fs
+    .accessSync(path)
+};
+
+const logger = (req, _, next) => {
+  try {
+    const start = performance.now();
+    next();
+
+    try {
+      isAccessible(pathLogDir)
```

Comment:
I don't like, that we make these checks for access on each logger call
Please restructure this method in following way
<img width="277" alt="image" src="https://user-images.githubusercontent.com/32495219/159351553-95486625-0c97-43af-8802-db9a5bdebeeb.png">